### DISCUSSION TITLE
Aerosol diagnostics on AeroVal

### DISCUSSION THREAD

**Discussion by andrearosendahl** [NorESMhub/noresm3_dev_simulations#60] [Show and tell]
Various aerosol variables for the NorESM2.5 simulations are now available on AeroVal, under the NorESM project. It includes maps, timeseries and statistics for the simulations, and can be compared to both CMIP6 and observations.

Two experiments are created as of now. Autumn24cases includes the simulations from August to November and NorESM2-MM (preindustrial). LatestNorESM2.5case includes many variables from the two newest simulations, NorESM2-MM (preindustrial) and a couple of extra models from present-day climate.  

Link to the website here:
https://aeroval.met.no/pages/evaluation/?project=noresm&experiment=LatestNorESM2.5case

**Comment by oyvindseland** [comment]
Thank you for the work.  Will be a good tool for analysing the results.
One minor issue. I can not find what wavelength for the the optical properties anywhere. I think this should be added. (Visible? 550 nm? )

**Reply by andrearosendahl** [reply]
Yes, it is 550nm. That's a good suggestion, I will add it for the next run!
